src/realm/bplustree.hpp

```diff
@@ -203,6 +203,10 @@ class BPlusTreeBase {
             m_root->bp_set_parent(parent, ndx_in_parent);
     }
 
+    virtual Mixed get_any(size_t) = 0;
```

Comment:
This should/could be const. `virtual Mixed get_any(size_t) const = 0`

Reply:
Sure